rakelib/sync.rake

```diff
@@ -0,0 +1,31 @@
+namespace :sync do
+  REPOSITORIES = [:StencilSwiftKit, :SwiftGenKit, :SwiftGen, :templates]
+
+  desc 'Synchronize all files across repositories'
+  task :all => [:rakelib, :gitignore, :license]
+
+  task :gems do |task|
+    REPOSITORIES.each do |repository|
+      FileUtils.cp('common/Gemfile', "#{repository}/")
+      FileUtils.cp('common/Gemfile.lock', "#{repository}/")
```

Comment:
If you don't specify anything, then `bundle install` installs the gems system-wide, so in your `$GEM_HOME`
There's a flag you can use when invoking `bundle install` to specify another path, like `./vendor` for example and gems would instead be installed here, and that path is stored in a local file by bundler so that it automatically reuses the same location for this local project on further invocations of `bundle install` or `bundle update` from this same directory

So indeed as we install our gems system wide invoking `bundle install` here won't do much. But (1) that doesn't hurt either (2) in case some future contributors decided to install their gems locally and not system wide it would allow that to work too.